File: MuziMusic/app/src/main/java/com/ahuynh/muzimusic/ui/component/artist/ArtistFragment.java
package com.ahuynh.muzimusic.ui.component.artist;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

import com.ahuynh.muzimusic.databinding.FragmentArtistBinding;


public class ArtistFragment extends Fragment {

    private FragmentArtistBinding binding;


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = FragmentArtistBinding.inflate(inflater, container, false);
        View view = binding.getRoot();
        return view;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: MuziMusicApp/app/src/main/java/com/ahuynh/muzimusicapp/ui/component/artist/ArtistFragment.kt
import android.os.Bundle
import android.view.View
import com.ahuynh.muzimusicapp.databinding.FragmentArtistBinding
import com.ahuynh.muzimusicapp.ui.base.BaseFragment
import dagger.hilt.android.AndroidEntryPoint


class ArtistFragment : BaseFragment<FragmentArtistBinding>(FragmentArtistBinding::inflate) {
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}